husky_bringup/launch/robotiq_s_model_grippers/robotiq_s_gripper_left.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
```

Comment:
Add a higher level launch file that does the include and parametrization, but DRY on the actual node launch and etc.